lib/reporters/xunit.js

```diff
@@ -130,7 +130,7 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === 'failed') {
     var err = test.err;
-    this.write(tag('testcase', attrs, false, tag('failure', {}, false, cdata(escape(err.message) + '\n' + err.stack))));
```

Comment:
@dasilvacontin you need to to do CDATA escaping properly the function should be:
```js
function cdata(content) {
    return `<!CDATA[${s.replace(']]>',']]]]><![CDATA[>')}]]>`;
}

while escape within CDATA is indeed "hurr, durr", it's safer than just wrapping in CDATA.